cmd/neofs-lens/internal/object/get.go

```diff
@@ -0,0 +1,38 @@
+package object
+
+import (
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/spf13/cobra"
+)
+
+var getObjectCMD = &cobra.Command{
+	Use:   "get",
+	Short: "Get object from NeoFS",
```

Comment:
not sure `from NeoFS` is a suitable wording, SN's storage engine?

Reply:
```suggestion
	Short: "Get object from the NeoFS node's storage snapshot",
```